kentikapi/internal/resources/base.go

```diff
@@ -126,3 +134,17 @@ func (b BaseAPI) DeleteAndValidate(ctx context.Context, url string, output inter
 
 	return nil
 }
+
+func LogPayload(l bool, msg, url string, payload interface{}) {
+	if l {
+		if payload == "" {
+			log.Printf("%s %s", msg, url)
+		} else {
+			jsonData, err := json.Marshal(&payload)
```

Comment:
There is no need for json.Marshal here. Just take JSON string as function parameter.

Reply:
Done